extensions/amp-story/1.0/amp-story-bookend.css

```diff
@@ -266,6 +268,43 @@
   margin-right: 32px !important;
 }
 
+.i-amphtml-story-bookend-cta-link-wrapper {
+  display: flex !important;
+  flex-wrap: wrap !important;
+  box-sizing: border-box !important;
+  color: inherit !important;
+  overflow: hidden !important;
+  flex: 1 0 320px !important;
+  justify-content: space-between !important;
+  margin: 24px 0 !important;
+  padding: 0 32px !important;
+}
+
+.i-amphtml-story-bookend-cta-link {
+  background-color: rgba(255, 255, 255, 0.15) !important;
+  border-radius: 100px !important;
+  overflow: hidden !important;
+  text-decoration: none !important;
+  width: 45% !important;
```

Comment:
On mobile these buttons don't allow for much text, so I think we should try to optimize the real estate, as well as using the same margins horizontally and vertically.

How about
````
width: calc(50% - 6px);
/* remove max-width since containers are never going to be super wide anyway */
margin-bottom: 12px;
````

(12px over 10px is super arbitrary, feel free to ignore, but I'd like to harmonize all the margins by using 12, 24, or 40px eventually).

Before:
![image](https://user-images.githubusercontent.com/1492044/40142896-f13d3662-5927-11e8-9aa6-89575646d8a9.png)

After:
![image](https://user-images.githubusercontent.com/1492044/40142898-f4cd3322-5927-11e8-9813-6d05fac654e2.png)

Reply:
Done.